### DISCUSSION TITLE
how to link the generated static lib to my visual studio project

### DISCUSSION THREAD

**Discussion by Ning-Li-Ambition** [scivision/mumps-superbuild#29] [General]
hi there,
I use visual studio on windows with intel oneAPI Fortran.
I have never used ninja before, but used cmake to install libraries on windows. my question is, if I follow your instructions, what will I get? A static library? And could I link it to my personal visual studio project?
Thanks

**Comment by scivision** [comment]
Visual Studio works to build MUMPS, but I haven't tried using the static lib files in a Visual Studio project.
From oneAPI command prompt on Windows:

```sh
cmake -B build -G Ninja  -DCMAKE_C_COMPILER=cl
```

```sh
cmake --build build
```

under build/ directory I get static libraries:

```
dmumps.lib
mumps_common.lib
pord.lib
smumps.lib
```

**Comment by scivision** [comment]
I documented this in a3c383f78608474c19f71adada3b8ff6e7305dcf
